getupio-undistro/undistro
# Fix second helm release installation

## Description
Helm Releases were not being installed in workload cluster on the second attempt.

## Linked Issues
There is no linked issues.

## How has this been tested?
- Apply a DefaultPolicies referencing a cluster (`kubectl apply -f defaultpolicies.yaml`);
- Delete the DefaultPolicies applied before (`kubectl delete -f defaultpolicies.yaml`);
- Apply it again (`kubectl apply -f defaultpolicies.yaml`);

In this example, kyverno is installed in workload cluster only on first step.

The expected behavior is that kyverno be installed in workload cluster on both attempts.

## Checklist
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/getupio-undistro/.github/labels?q=Type%3A)
- [x] My changes are covered by e2e tests
